scripts/start_dev_db.sh

```diff
@@ -1,16 +1,18 @@
 # This script is meant to be run with `source` so it can set environment variables.
 
-export PGDATA="$PWD/dev_pgdata"
-export PGHOST=$PWD
+export PGDATA="$PWD/target/dev_pgdata"
+export PGHOST="$PWD/target"
 
 # Necessary to encode the dev db path into proper URL params
 export ENCODED_HOST=$(printf $PWD | jq -sRr @uri)
 
 export PGUSER=postgres
-export DATABASE_URL="postgresql://lemmy:password@$ENCODED_HOST/lemmy"
+export DATABASE_URL="postgresql://lemmy:password@$ENCODED_HOST/target/lemmy"
```

Comment:
Instead of changing this, change encoded_host to reference pghost instead of pwd

Remove pushd and popd, since the psql and pg_ctl commands use these environment vars for paths, and the vars have absolute paths

Reply:
There's another reference to pwd in config_args that must be replaced with pghost